synapse/storage/relations.py

```diff
@@ -51,16 +130,26 @@ def get_relations_for_event(
                 type, if given.
             event_type (str|None): Only fetch events with this event type, if
                 given.
+            aggregation_key (str|None): Only fetch events with this aggregation
+                key, if given.
             limit (int): Only fetch the most recent `limit` events.
             direction (str): Whether to fetch the most recent first (`"b"`) or
                 the oldest first (`"f"`).
+            from_token (RelationPaginationToken|None): Fetch rows from the given
```

Comment:
looks like it's expecting a str rather than a token (which feels like a bit of a questionable decision, tbh - parsing the string doesn't feel like a thing  that belongs in the `storage` layer)